contrib/babelfishpg_tsql/sql/sys_functions.sql

```diff
@@ -987,6 +987,18 @@ END;
 $BODY$
 LANGUAGE plpgsql PARALLEL SAFE IMMUTABLE RETURNS NULL ON NULL INPUT;
 
+CREATE OR REPLACE FUNCTION sys.atn2(IN x DOUBLE PRECISION, IN y DOUBLE PRECISION) RETURNS DOUBLE PRECISION
+AS
+$BODY$
+DECLARE
+    res DOUBLE PRECISION;
+BEGIN
+    res = atan2(x, y);
+    RETURN res;
+END;
+$BODY$
+LANGUAGE plpgsql PARALLEL SAFE IMMUTABLE RETURNS NULL ON NULL INPUT;
```

Comment:
It keeps throwing syntax error `~~ERROR (Message: syntax error at or near "SYS")~~` after I changed the language from `plpgsql` to `sql`... I tried change 1)nothing but the language, 2)every above suggested changes including changing the language, 3) one of above suggested change at a time along with changing the language, and they all throw syntax errors...

No error is thrown when not changing the language (but change everything else).

Reply:
This would work
```
CREATE OR REPLACE FUNCTION sys.atn2(IN x sys.float, IN y sys.float)
RETURNS sys.float
AS
$$
    SELECT pg_catalog.atan2(x, y);
$$
LANGUAGE SQL PARALLEL SAFE IMMUTABLE RETURNS NULL ON NULL INPUT;
```